opengemini-client-api/src/main/java/io/opengemini/client/api/OpenGeminiAsyncClient.java

```diff
@@ -108,6 +108,14 @@ public interface OpenGeminiAsyncClient extends AutoCloseable {
      */
     CompletableFuture<Void> write(String database, String retentionPolicy, List<Point> points);
 
+    /**
+     * Writing via GRPC points to the database.
+     *
+     * @param database the name of the database.
+     * @param points   the points to write.
+     */
+    CompletableFuture<Void> writeByGrpc(String database, List<Point> points);
```

Comment:
The parameter is modified to`(WriteRequest writeRequest)`,referring to the definition in the Go SDK.